### PR TITLE
ADM-450 [Metrics] Active 'Next' button in metric page

### PR DESCRIPTION
**Pull request by tingwang1234**
## Summary
Only when all mandatory data in metrics page is filled, Next button is active, and users can click it to step into report page
card link: https://trello.com/c/Ko2O9LUV

## Before
image
Next button is always active

## After
image
image
Next button is active only when all mandatory data in metrics page is filled
 
## Note

_Null_

### CODE REVIEW

**Review by wangwang-zhang — APPROVED** [review]
LGTM

**Review by fengxinhou — APPROVED** [review]
LGTM

File: frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx [hunk 1]
```diff
@@ -13,38 +13,27 @@ interface realDoneProps {
...
-  return filteredStatuses
-}
+) => columns.filter(({ value }) => selectedDoneColumns.includes(value.name)).flatMap(({ value }) => value.statuses)
 
-function getDoneStatuses(columns: { key: string; value: { name: string; statuses: string[] } }[]) {
-  const doneStatuses =
-    columns.find((column) => column.key === METRICS_CONSTANTS.doneKeyFromBackend)?.value.statuses ?? []
-  return doneStatuses
-}
+const getDoneStatuses = (columns: { key: string; value: { name: string; statuses: string[] } }[]) =>
```
Comment (guzhongren) [inline frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx:24]: Statuses?

**Review by guzhongren — APPROVED** [review]

File: frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx (hunk 1 above)
Comment (tingwang1234) [reply frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx:24]: Sorry, it should be Status, I'll fix it right now

**Pull request merged**